resources/views/activity.blade.php

```diff
@@ -1,6 +1,15 @@
 @extends("layout.public")
 
+@section("title", $activity->title)
+@section("description", $activity->subtitle)
+
 @section("content")
+    <meta property="og:url"                content="https://blumilk.pl/pl/aktywnosci/{{ $activity->slug }}" />
```

Comment:
what about `en`?

Reply:
And what about these multiplicated spaces between attributes?

Reply:
Added universal route, removed spaces